README.md

```diff
@@ -21,37 +21,41 @@ gulp.task('minify', function() {
 });
 ```
 
-See the [html-minifier docs](https://github.com/kangax/html-minifier) for options.
+See the [html-minifer docs](https://github.com/kangax/html-minifier) for options.
```

Comment:
good catch. looks like it's been here since the beginning https://github.com/jonschlinkert/gulp-htmlmin/commit/ccc4d1e5f9eb6e28344d7e18042051ee40fca280#diff-8191924d78e7652d71643c49ac2716c8R20